smart-metastore/src/main/java/org/smartdata/metastore/MetaStore.java

```diff
@@ -1116,4 +1120,44 @@ public void insertBackUpInfo(BackUpInfo backUpInfo) throws MetaStoreException {
       throw new MetaStoreException(e);
     }
   }
+
+  public List<SystemInfo> listAllSystemInfo() throws MetaStoreException {
+    try {
+      return systemInfoDao.getAll();
+    } catch (Exception e) {
+      throw new MetaStoreException(e);
+    }
+  }
+
+  public List<SystemInfo> getSystemInfoByProperty(String property) throws MetaStoreException {
```

Comment:
Shouldn't the key `property` be exclusive? Better to just return a SystemInfo, not a list.